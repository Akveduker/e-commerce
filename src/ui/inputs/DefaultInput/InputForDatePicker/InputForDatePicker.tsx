import React, { forwardRef } from 'react';
import s from '../DefaultInput.module.scss'
const InputForDatePicker = forwardRef((props: any, ref) => {
    return <input {...props} ref={ref} className={s['input']} />;
})

export default InputForDatePicker;