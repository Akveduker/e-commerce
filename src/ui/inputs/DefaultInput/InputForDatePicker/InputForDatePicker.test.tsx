import { render } from "@testing-library/react"
import InputForDatePicker from "./InputForDatePicker"

describe('InputForDatePicker', () => {
    test('Снапшот', () => {
        const { container } = render(<InputForDatePicker />)
        expect(container).toMatchSnapshot()
    })
})