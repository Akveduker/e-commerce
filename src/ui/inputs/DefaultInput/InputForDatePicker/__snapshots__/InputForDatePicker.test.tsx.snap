// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputForDatePicker Снапшот 1`] = `
<div>
  <input
    class="input"
  />
</div>
`;
